src/components/elements/input/input.js

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import { InputStyle, ErrorText, Label } from './input-style'
 import { VFlex } from 'components/elements'
 
-export const Input = ({ error, className, name, ...props }) => {
+export const Input = ({ error, className, id, ...props }) => {
     return (
         <VFlex className={className}>
-            <Label htmlFor="email">{name}</Label>
```

Comment:
This adds an email label. in the design there is no label is that fine?

Reply:
ive added display: none to the label, so it doesnt show in the UI. label is necessary to improve accessibility score